pkg/server/plugin/datastore/sql/sql_test.go

```diff
@@ -321,9 +321,29 @@ func (s *PluginSuite) TestBundleCRUD() {
 	s.Require().NoError(err)
 	s.AssertProtoEqual(bundle3, anresp.Bundle)
 
-	// update
+	// update with mask
 	expectedCallCounter = ds_telemetry.StartUpdateBundleCall(s.expectedMetrics)
 	uresp, err := s.ds.UpdateBundle(ctx, &datastore.UpdateBundleRequest{
+		Bundle: bundle,
+		InputMask: &common.BundleMask{
```

Comment:
This test isn't really testing that the mask was actually applied since it sets all the fields to `true`, which would be indistinguishable from the mask being ignored...